### DISCUSSION TITLE
[Q&A] Issue Running App Locally with GitHub Codespaces -Metadata.metadataBase Error

### DISCUSSION THREAD

**Discussion by MEClouds** [lobehub/lobehub#1033] [Development Question] [answered]
**Description:**
When attempting to run the application locally using GitHub Codespaces, I encountered the following issue related to `metadata.metadataBase`:

```
⚠ metadata.metadataBase is not set for resolving social open graph or twitter images, using "http://localhost:3010". See https://nextjs.org/docs/app/api-reference/functions/generate-metadata#metadatabase
```

**Steps to Reproduce:**
1. Clone the repository.
2. Click on 
![image](https://github.com/lobehub/lobe-chat/assets/84941140/c5f5fa24-79a8-468d-99b9-b12355415a28)
3. Run the application locally using GitHub Codespaces.

**Expected Behavior:**
The application should run without any metadata-related warnings.

**Actual Behavior:**
The warning mentioned above is displayed in the console.

**Screenshots:**
![image](https://github.com/lobehub/lobe-chat/assets/84941140/87ba7e30-b4f6-4a1b-bd23-78be952c2449)

![image](https://github.com/lobehub/lobe-chat/assets/84941140/919df5e4-cc3d-4d9f-a539-413b9ec206df)

**Comment by arvinxx** [comment, marked as answer]
It's not an error. just a warning
